docs/src/quickstart/hello_klio_batch.rst

```diff
@@ -0,0 +1,127 @@
+
+
+:violetemph:`Next`, within your project directory, run the following command:
+
+.. code-block:: sh
+
+    $ klio job create --job-name klio-quick-start --job-type batch --create-resources --use-defaults
+
+
+After responding to the prompts, Klio will:
+
+1. Create a GCS bucket in the provided GCP project you provided in the prompt for output data: ``gs://$GCP_PROJECT-output``.
+2. Create a `Google Stackdriver`_ dashboard in the provided GCP project for you to monitor runtime job metrics.
+3. Create required files within the current working directory.
+
+:violetemph:`Then`, commit the created job files into ``git``:
+
+.. code-block::
+
+    $ git add .
+    $ git commit -m "Initial commit for Klio quickstart example"
+
+
+Run the New Klio Job
+--------------------
+
+.. caution::
+
+    If the |klio-cli|_ was installed via :ref:`option 2 <install-option-2>` or :ref:`option 3 <install-option-3>`, make sure you're ``klio-cli`` virtualenv is activated.
+
+:violetemph:`First`, add a file of IDs elements as text input.
+
+The default event input points to a text file in ``klio-quick-start_input_ids.txt`` in a GCS bucket.
+To get testing quick this file event input should be replaced with a local file.
+A local file containing lines of text corresponding to elements can be created manually
+in the top level job directory for this quickstart. Assuming a linux machine, a local file can
+also be created by running the following command while in the klio job directory.
+
+.. code-block::
+
+    $ { echo hello
+        echo world
+      } > klio-quick-start_input_ids.txt
+
+This will create a local file with two lines of text that will serve as the event inputs
+of the batch Klio job.
+
+:violetemph:`Next`, adjust event input config ``klio-job.yaml``:
+
+    .. code-block:: yaml
+       :emphasize-lines: 9,13
+
+        name: klio-quick-start
+        pipeline_options:
+          streaming: False
+        <-- snip -->
+        job_config:
+          event:
+            inputs:
+              - type: file
+                location: ./klio-quick-start_input_ids.txt
+          data:
+            inputs:
+              - type: gcs
+                skip_klio_existence_check: True
+
+Note the ``job_config.event.inputs`` field of location is now set to the local file that was created
+and the ``job_config.data.inputs`` field ``skip_klio_existence_check`` was set to ``True``. This is
+to skip default existence checks while getting started.
+
+:violetemph:`Then`, run the job using DirectRunner:
+
+Klio will first build a Docker image of the example job with the required dependencies, then start the job locally.
+To know it started successfully, you should see a log line containing
+
+.. code-block:: text
+
+    Running pipeline with DirectRunner
+
+
+Klio will then read from the event input, in this case, the text file that was created.
+A :ref:`Klio message<klio-message>` is created for each element in the file
```

Comment:
Maybe instead of "...is created for each element in the file...", say "...is created for each line in the file..."?